src/lib/config/default/config-sap-passport.json

```diff
@@ -7,101 +7,100 @@
                 "fieldName": "sap-passport"
             },
             "output": [
-                "msg-log",
```

Comment:
why is this not possible in msg-logs?

Reply:
clarified in call: we will keep this for now.